src/ResourceManager/Sql/Commands.Sql/ManagedInstance/Cmdlet/UpdateAzureSqlManagedInstance.cs

```diff
@@ -0,0 +1,215 @@
+
+        /// <summary>
+        /// Gets or sets the name of the resource group to use.
+        /// </summary>
+        [Parameter(ParameterSetName = UpdateByNameAndResourceGroupParameterSet,
+            Mandatory = true,
+            Position = 1,
+            HelpMessage = "The name of the resource group.")]
+        [ResourceGroupCompleter]
+        [ValidateNotNullOrEmpty]
+        public override string ResourceGroupName { get; set; }
+
+        /// <summary>
+        /// The new SQL administrator password for the Managed instance.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The new SQL administrator password for the Managed instance.")]
+        [ValidateNotNull]
+        public SecureString AdministratorPassword { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Managed instance License Type
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Determines which License Type of Sql Azure Managed Instance to use")]
+        [PSArgumentCompleter(Constants.LicenseTypeBasePrice, Constants.LicenseTypeLicenseIncluded)]
+        public string LicenseType { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Storage Size in GB for Managed instance
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Determines how much Storage size to associate with Managed instance")]
+        public int? StorageSizeInGB { get; set; }
+
+        /// <summary>
+        /// Gets or sets the VCore for Managed instance
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Determines how much VCore to associate with Managed instance")]
+        public int? VCore { get; set; }
+
+        /// <summary>
+        /// The tags to associate with the Managed instance.
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "The tags to associate with the Managed instance.")]
+        [Alias("Tags")]
+        public Hashtable Tag { get; set; }
+
+        /// <summary>
+        /// Gets or sets whether or not to assign identity for Managed instance
+        /// </summary>
+        [Parameter(Mandatory = false,
+            HelpMessage = "Generate and assign an Azure Active Directory Identity for this Managed instance for use with key management services like Azure KeyVault.")]
+        public SwitchParameter AssignIdentity { get; set; }
+
+        /// <summary>
+        /// AzureSqlManagedInstanceModel object to remove
+        /// </summary>
+        [Parameter(ParameterSetName = UpdateByInputObjectParameterSet,
+            Mandatory = true,
+            Position = 0,
+            ValueFromPipeline = true,
+            HelpMessage = "The AzureSqlManagedInstanceModel object to remove")]
+        [ValidateNotNullOrEmpty]
+        [Alias("ManagedInstance")]
+        public Model.AzureSqlManagedInstanceModel InputObject { get; set; }
+
```

Comment:
Move Inputobject and ResoruceId parameters to the top so they will appear first in the help file.